### PR TITLE
[TEVA-1235] Update service start page banner

### PR DESCRIPTION
**Pull request by cpjmcquillan**
This PR implements some of the updates to the service start page.

## Jira ticket URL
https://dfedigital.atlassian.net/browse/TEVA-1235

## Changes in this PR:
- Service start page updates

### Before
image

### After
image

### CODE REVIEW

File: app/frontend/styles/components/search_panel.scss [hunk 1]
```diff
@@ -1,10 +1,33 @@
...
+  left: 50%;
+  margin-bottom: govuk-spacing(6);
+  margin-left: -50vw;
+  padding-bottom: govuk-spacing(3);
+  padding-top: govuk-spacing(1);
+  position: relative;
+  width: 100vw;
+}
+
+body {
```
Comment (alexbowen) [inline app/frontend/styles/components/search_panel.scss:12]: why the `body` css selector? is `.pages_home` selector on it on not enough?

File: config/locales/en.yml [hunk 2]
```diff
@@ -205,9 +205,9 @@ en:
         description_html: <p>We’re completing planned maintenance on job listings.</p><p>You can search and view vacancies, but you can’t manage them.</p><p>You’ll be able to use the full service again in less than an hour.</p><p><a href='mailto:teaching.vacancies@education.gov.uk'>Contact our support team</a> if you need help.</p>
       signin:
         title: List a teaching job
-        description: You must sign in to your Teaching Vacancies account to list a job at your school.
-        signin_link: how to get one
-        signin_guidance_html: Don't have an account? Find out %{link} if your school is eligible.
+        description: Sign in to your Teaching Vacancies account to list a job at your school or Trust.
```
Comment (alexbowen) [inline config/locales/en.yml:208]: Trust should be non caps

File: config/locales/en.yml [hunk 3]
```diff
@@ -347,7 +347,10 @@ en:
       labels:
         action_required: action required
         new: new
-    heading: 'Find a job in teaching'
+    heading: Welcome to Teaching Vacancies
+    intro: >-
+      Teaching Vacancies is a free job-listing service from the Department for Education.
+      You can search for a job at a school or Trust in England and set up job alerts.
```
Comment (alexbowen) [inline config/locales/en.yml:353]: "

File: config/locales/en.yml (hunk 2 above)
Comment (cpjmcquillan) [reply config/locales/en.yml:208]: @valentine-carter says NO!!

File: app/frontend/styles/components/search_panel.scss (hunk 1 above)
Comment (cpjmcquillan) [reply app/frontend/styles/components/search_panel.scss:12]: I just copypastad 🍝  the css block for a different controller class to here, but in fact in a subsequent PR I apply zero padding to `.govuk-main-wrapper` everywhere, so I will just do that in this PR instead

**Review by alexbowen — APPROVED** [review]

**Pull request merged**